src/connection_string.ts

```diff
@@ -1062,7 +1062,10 @@ export const OPTIONS = {
   passphrase: { type: 'any' },
   pfx: { type: 'any' },
   secureProtocol: { type: 'any' },
-  index: { type: 'any' }
+  index: { type: 'any' },
+  // Legacy Options, these are unused but left here to avoid errors with CSFLE lib
+  useNewUrlParser: { type: 'boolean' } as OptionDescriptor,
+  useUnifiedTopology: { type: 'boolean' } as OptionDescriptor
```

Comment:
And here that's supporting the old options. These still aren't listed in the props for MongoClientOptions (which is why we have the `as` casting). I would add that they are deprecated but then our CSFLE lib would emit the warnings with no way to disable.